chapter2/tests/t7/chapter2-l2-t7-get-unique-files.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Please fix the same moments as in https://github.com/krasninja/devops-camp-2023/pull/6 PR

Reply:
1. Add dir input arg check.
2. Show errors of file command
3. Use global "dir".
4. Use `sort -u`.